scripts/build.sh

```diff
@@ -1,2 +1 @@
-node_modules/.bin/babel lib/ --out-dir dist
-node_modules/.bin/coffee -co dist/ lib/
+babel lib/ --out-dir dist/ --presets es2015,stage-0,react
```

Comment:
My limited experience in this matter has led me to take an over cautious approach here but given what you are saying, I'll adjust this to reflect reliance on .babelrc file instead.